crates/fuel-core/src/executor.rs

```diff
@@ -458,6 +480,7 @@ where
         Ok(data)
     }
 
+    #[allow(clippy::too_many_arguments)]
```

Comment:
Hmm, maybe it is better to create `ExecutorContext` that contains all data(like header, execution data, tx, etc) and implement all methods directly to this context. Because it seems we propagate a lot of the same object.

But it's okay to do that in follow-up PR, if you don't want to do that now=)

Reply:
Yeah I'll need to think on this, since each of these fn's has slightly different params. Although maybe there's a way to combine a few of them together.